src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/GoToDefinitionTests.cs

```diff
@@ -131,31 +131,37 @@ public async Task GoToDefinition_ComponentAttribute_Minimized()
         await TestServices.Editor.WaitForCurrentLineTextAsync("public bool MyProperty { get; set; }", ControlledHangMitigatingCancellationToken);
     }
 
-    [IdeFact(Skip = "https://github.com/dotnet/razor/issues/8036")]
+    [IdeFact]
     public async Task GoToDefinition_ComponentAttribute_InCSharpFile()
     {
         // Create the files
+        const string MyComponentPath = "MyComponent.cs";
         await TestServices.SolutionExplorer.AddFileAsync(RazorProjectConstants.BlazorProjectName,
-            "MyComponent.cs",
+            MyComponentPath,
             """
             using Microsoft.AspNetCore.Components;
             
             namespace BlazorProject;
 
             public class MyComponent : ComponentBase
             {
-                [Parameter] public string MyProperty { get; set; }
+                [Parameter] public string? MyProperty { get; set; }
             }
             """,
             cancellationToken: ControlledHangMitigatingCancellationToken);
 
+        await TestServices.SolutionExplorer.BuildSolutionAndWaitAsync(ControlledHangMitigatingCancellationToken);
+        await TestServices.Workspace.WaitForProjectSystemAsync(ControlledHangMitigatingCancellationToken);
+
+        const string MyPagePath = "MyPage.razor";
```

Comment:
Nit: That this is not at the top of the method with the other one causes me anxiety :P